jbpm-case-mgmt/jbpm-case-mgmt-impl/src/test/filtered-resources/META-INF/persistence.xml

```diff
@@ -76,9 +76,10 @@
     
     <!-- error handling -->
     <class>org.jbpm.runtime.manager.impl.jpa.ExecutionErrorInfo</class>
-    
+
     <properties>
-      <property name="hibernate.dialect" value="org.hibernate.dialect.PostgreSQLDialect" />
+      <property name="hibernate.dialect" value="${maven.hibernate.dialect}" />
+      <property name="hibernate.default_schema" value="${maven.jdbc.schema}"/>
```

Comment:
This is missing in the `jbpm-human-task-audit` persistence.xml and others in this PR. Is it even needed? But if it works as is, it's fine for me.

Reply:
Nice catch.
Seems it might be used depending on the db we're using - see https://github.com/kiegroup/jbpm/blob/master/jbpm-container-test/jbpm-remote-ejb-test/jbpm-remote-ejb-test-app/src/main/resources/META-INF/persistence.xml#L64

As per [hibernate doc](https://docs.jboss.org/hibernate/orm/5.3/userguide/html_single/Hibernate_User_Guide.html#configurations-mapping), it's used for qualifying tables in queries... so I will keep them and add it where it's missing just in case

Also I notice below properties in `datasource.properties` files are not used anywhere in the code, so I'm removing them:
- defaultSchema
- makeBaseDb
- testMarshalling